compiler/mx.compiler/mx_graal_benchmark.py

```diff
@@ -784,6 +784,163 @@ def benchSuiteName(self):
 
 mx_benchmark.add_bm_suite(DaCapoMoveProfilingBenchmarkSuite())
 
+class DaCapoD3SBenchmarkSuite(DaCapoBenchmarkSuite): # pylint: disable=too-many-ancestors
+    """DaCapo 9.12 Bach benchmark suite implementation with D3S modifications."""
+
+    def name(self):
+        return "dacapo-d3s"
+
+    def daCapoSuiteTitle(self):
+        return "DaCapo 9.12-D3S-20180206"
+
+    def daCapoClasspathEnvVarName(self):
+        return "DACAPO_D3S_CP"
+
+    def daCapoLibraryName(self):
+        return "DACAPO_D3S"
+
+    def successPatterns(self):
+        return ['']
```

Comment:
I'm not sure that you should to override this. Does the standard output change in this suite?

If you do want to change the success patterns, then you would need to return a list of regex patterns here (see other `BaseDaCapoBenchmarkSuite` as an example).

Alternatively, if there is no success pattern, it should just be an empty list.

Reply:
The standard output changed but it makes sense here to check for failures only. Should be fixed now.